app/travis_job.rb

```diff
@@ -4,7 +4,7 @@
 
 module Tint
 	class TravisJob
-		QUEUE_DIR = Pathname.new(ENV.fetch("TRAVIS_WORKER_BASE_DIR")).realpath
+		QUEUE_DIR = Pathname.new(ENV["TRAVIS_WORKER_BASE_DIR"].to_s).realpath
```

Comment:
Is `to_s` necessary? I think everything coming out of the environment will be a string, no?